config/config.go

```diff
@@ -542,3 +555,38 @@ func getDockerCID() (string, error) {
 type proxyProb struct {
 	Proxies yaml.MapSlice
 }
+
+type AppTLS struct {
```

Comment:
I would have named it just `TLS` and put `cert_path` and `key_path` directly under it. If TLS is configured then both gRPC and HTTP interface should use it. It is ok if this PR only enables them for gRPC, later we can make HTTP use it too. If feel like implementing it right here that would be great.

Reply:
sounds good, i'll push up the changes for that as well

Reply:
:ping_pong: with feedback. the changes to the proxy are a little ugly, but it's not too bad, imo